torch/csrc/jit/test_jit.cpp

```diff
@@ -496,7 +435,122 @@ void interpStageTest() {
     JIT_ASSERT(exactlyEqual(outputs[1],cx));
 }
 
+using var_meta_type = std::vector<int64_t>;
```

Comment:
nit: var_size_spec or something like that "meta_type" suggests some something more magical than this is :)